internal/collector/metrics.go

```diff
@@ -296,7 +297,7 @@ type QuotaPluginMetricsInstance struct {
 func (c *QuotaPluginMetricsCollector) Describe(ch chan<- *prometheus.Desc) {
 	quotaPluginMetricsOkGauge.Describe(ch)
 	for _, plugin := range c.Cluster.QuotaPlugins {
-		plugin.DescribeMetrics(ch)
+		plugins.LiquidDescribeMetrics(ch, plugin.(*plugins.LiquidQuotaPlugin).LiquidServiceInfo.UsageMetricFamilies, []string{"domain_id", "project_id"})
```

Comment:
The QuotaPlugin interface already has a method for getting the ServiceInfo:
```suggestion
		plugins.LiquidDescribeMetrics(ch, plugin.ServiceInfo().UsageMetricFamilies, []string{"domain_id", "project_id"})
```